doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -634,6 +634,88 @@ must be sent.
 *default location*, then the query will be declared as ambigous and an HTTP error response with code ``409``
 must be sent. 
 
+## Notification messages
+
+Notifications include two fields:
+
+* `subscriptionId` which represents the concerned subscription that originates the notification
+* `data` an array with the notification data itself which includes the entity and attributes concerned. Each
+  element in the array corresponds to a different entity. The way of representing each entity depends on
+  the `attrsAs` parameter associated to the subscription, as shown below.
+
+
+If `attrsAs` is `normalized` then default entity representation is used:
+
+```
+{
+   "subscriptionId": "12345",
+   "data": [
+     {
+       "id": "Room1",
+       "type": "Room",
+       "temperature": {
+         "value": 23,
+         "type": null,
+         "metadata": {}
+       },
+       "humidity": {
+         "value": 70,
+         "type": "percentage",
+         "metadata": {}
+       }
+     },
+     {
+     "id": "Room2",
+     "type": "Room",
+     "temperature": {
+       "value": 24,
+       "type": null,
```

Comment:
does the current version represent null attribute types as 'none' or as 'null' ?